pkgs/development/python-modules/solo-python/default.nix

```diff
@@ -14,15 +14,15 @@
 
  buildPythonPackage rec {
   pname = "solo-python";
-  version = "0.0.27";
+  version = "0.0.30";
   format = "flit";
   disabled = pythonOlder "3.6"; # only python>=3.6 is supported
 
   src = fetchFromGitHub {
     owner = "solokeys";
     repo = pname;
     rev = version;
-    sha256 = "sha256-OCiKa6mnqJGoNCC4KqI+hMw22tzhdN63x9/KujNJqcE=";
+    sha256 = "0hr4w33m5kf23vyq6advlmyhjqp3i7p34y6iqy61mc06a0zxq9yv";
```

Comment:
I'm getting a mismatch with `nix-review`:

```bash
hash mismatch in fixed-output derivation '/nix/store/zzi3z0r84kf7izn40piq675n0lzrz5nx-source':
  wanted: sha256:0hr4w33m5kf23vyq6advlmyhjqp3i7p34y6iqy61mc06a0zxq9yv
  got:    sha256:1i9kybp08qfcdx6m4wl7ij40y1v17mvvhcdg7zglwfakblf69w41
```

Reply:
You are right, my override had `fetchPypi`. updating.